book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  # @param [String] title
+  # @param [String] author
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] In this Book class kindly add the method for the has-many-through relationship with the person class using the Rental class.

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```